app/services/partnership_notification_service.rb

```diff
@@ -43,6 +43,45 @@ def send_reminder(partnership)
     end
   end
 
+  # MOOSE
+  def send_invite_sit_reminder_to_partnered_schools
+    current_year = Cohort.current.start_year
+
+    School.joins(:school_cohorts)
+      .partnered(current_year)
+      .where(school_cohorts: { induction_programme_choice: %i[full_induction_programme] })
+      .where.missing(:induction_coordinators)
+      .includes(:partnerships)
+      .find_each do |school|
+        ActiveRecord::Base.transaction do
+          partnership = school.partnerships.unchallenged.in_year(current_year).first
+
+          notification_email = create_notification_email(
+            partnership,
+            PartnershipNotificationEmail.email_types[:nominate_sit_email],
+          )
+
+          nomination_email = NominationEmail.create_nomination_email(
+            sent_at: notification_email.created_at,
+            sent_to: notification_email.sent_to,
+            school: notification_email.partnership.school,
+            partnership_notification_email: notification_email,
+          )
+
+          notify_id = SchoolMailer.partnered_school_invite_sit_email(
+            recipient: notification_email.sent_to,
+            lead_provider_name: notification_email.lead_provider.name,
+            delivery_partner_name: notification_email.delivery_partner.name,
+            nominate_url: nomination_email.nomination_url(utm_source: :partnered_invite_sit_reminder),
+            challenge_url: challenge_url(notification_email.token, utm_source: :partnered_invite_sit_reminder),
+          ).deliver_now.delivery_method.response.id
```

Comment:
Yes because we should set the notify id on the db record. This is the same as the other partnership notification email methods, I think its fine as we are doing this ourselves in the console, its not going to break any user flow if it fails.

Reply:
Ah, okay. We are capturing that we've sent it so we can re-run this safely.